src/github.com/getlantern/yamlconf/yamlconf.go

```diff
@@ -103,6 +103,14 @@ type Manager struct {
 	// example for fetching config updates from a remote server.
 	CustomPoll func(currentCfg Config) (mutate func(cfg Config) error, waitTime time.Duration, err error)
 
+	// Obfuscate: if true, the on-disk version of the config will be obfuscated by
+	// encrypting it using AES-128 encryption.
```

Comment:
Wow, this actually turns out to make a big difference.  Without encryption we're looking at about 300ms to read the config, with encryption about 1.6 seconds!  I'll do a custom ROT13 thing.